pkg/stacks/relationship_test.go

```diff
@@ -0,0 +1,310 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stacks
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	namespace             = "cool-namespace"
+	uidString             = "definitely-a-uuid"
+	resourceName          = "cool-resource"
+	thirtyTwoAs           = strings.Repeat("a", 32)
```

Comment:
Nitpick: originally I read it as the word `as`

Reply:
I could come up with another character that can't be confused as a word or abbreviation in this context, but I would also have to update all of the suffixes.  I could add `Letter` to the variable name, but it becomes lengthy.

I think looking at the assignment disambiguates the meaning of the name of this limited use testing variable.